app/controllers/admin/secrets_controller.rb

```diff
@@ -58,7 +60,11 @@ def project_permalink
 
   def successful_response(notice)
     flash[:notice] = notice
-    redirect_to action: :index
+    if params[:commit] == ADD_MORE
+      redirect_to new_admin_secret_path(secret: params[:secret].except(:value))
```

Comment:
yeah ... was thinking that too ... but then I thought they might be adding the same key to multile envs/deploy-groups ... so I opted for keeping it